apollo-http-cache/src/main/kotlin/com/apollographql/apollo3/cache/http/CachingHttpInterceptor.kt

```diff
@@ -128,12 +131,14 @@ class CachingHttpInterceptor(
     return response
   }
 
+  @Deprecated("Use store.clearAll() instead", ReplaceWith("store.clearAll()"))
```

Comment:
Nitpick: let's add the `@ApolloDeprecatedSince` annotation to keep track of these (`v3_2_0` needs to be added to the enum).

Reply:
Good call 👍 . I went with `v3_1_1` so we're not stuck on the versioning scheme and may decided to ship either `3.1.1` or `3.2.0`